### ISSUE TITLE
[FEATURE-REQUEST] add ping endpoint

### ISSUE THREAD

**Issue by FingerLiu** [devopsfaith/krakend#181]
When deploying krakend in production, it(lb before krakend or simple k8s deployment) requires krakend to have a /ping endpoint to do health check things of krakend it self.
For now, I make a /ping backend and pass it to /ping endpoint.
This way can solve this problem, but I think this should be done by krakend it self.
Any suggestions ?

**Comment by alombarte** [comment]
@FingerLiu updated my previous comment with extended info

**Comment by alombarte** [comment]
Added in the documentation: https://www.krakend.io/docs/endpoints/health/

**Comment by jharley** [comment]
Apologies for commenting on a stale thread. I have followed the [health endpoint](https://www.krakend.io/docs/endpoints/health/) document, but because `fake-backend` doesn't resolve my logs are full of resolution errors which is hardly ideal :\ Is there a recommended way around this?

**Comment by alombarte** [comment]
Hi @jharley , no problem!
The documentation refers to several ways of achieving it, although certainly there is only one example. One of the options is that you use KrakenD itself as a backend via its `/__debug/` endpoint, so you don't refer to any fake backend.

An important thing about this system is that you have to **make sure to start the server with `-d` flag**.
Enabling the debug flag in production is **perfectly safe**, as it only exposes a dummy `/__debug` endpoint that returns a ping-pong answer. No debug information is returned whatsoever, so use it without any fear.

For instance:
```{
        "endpoint": "/health",
        "comment": "This health endpoint requires to start the server in production with the -d flag (yeah, it's safe)",
        "extra_config": {
          "github.com/devopsfaith/krakend/proxy": {
            "static": {
              "data": {
                "status": "OK"
              },
              "strategy": "always"
            }
          }
        },
        "backend": [
          {
            "url_pattern": "/__debug/health",
            "host": [
              "http://localhost:{{ .env.port }}"
            ]
          }
        ]
      }
```

**Comment by alombarte** [comment]
The `/__health` endpoint is now available by default (since 1.2).
https://www.krakend.io/docs/endpoints/health/
